apps/sms/views/shared/js/utils.js

```diff
@@ -940,7 +940,12 @@
         });
 
       return promise.then(
-        () => shimHostIframe.contentWindow.bootstrap(appInstanceId)
+        () => {
+          window.mozSetMessageHandler = function(type, handler) {
```

Comment:
that works for me, but please add a reference to the bug # + some explanation in a comment